src/client/internal.ts

```diff
@@ -24,6 +24,14 @@ export const SERVICE_IDENTITY = {
     name: 'fdc3-service'
 };
 
+/**
+ * The identity of the resolver window, running on the FDC3 service application
+ */
+export const RESOLVER_IDENTITY: Identity = {
```

Comment:
Makes sense, it's only used in there and integration tests. 

Is it OK to export the const from `ResolverHandler.ts` so it can be imported from the test file, or just make a copy of it in `test\demo\constants.ts`?

Reply:
Copy is fine, so long as it's only in the one place in the tests.

Current setup looks good